frontend/src/components/FormTextarea.tsx

```diff
@@ -0,0 +1,38 @@
+import React from 'react';
+import Textarea, { TextareaProps } from './Textarea';
+import { classNames } from '../common/utils/utils';
+import FormReadOnlyElement from './FormReadOnlyElement';
+
+interface FormTextareaProps extends TextareaProps {
+  errorMessage?: string;
+}
+
+const FormTextarea = ({
+  errorMessage,
+  readOnly,
+  defaultValue,
+  label,
+  className,
+  ...props
+}: FormTextareaProps) => {
+  return readOnly ? (
+    <FormReadOnlyElement label={label} value={defaultValue || ''} />
+  ) : (
+    <div className="flex flex-col gap-1">
+      <Textarea
```

Comment:
Please add the helper to the Textarea component, same as it is for the Input component. I would like the textarea and input component to follow the same structure. The same should apply for FormTextarea and FormInput